CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Can we add it to the .github folder?

Reply:
@IgorGov the right place for this file is the root directory as far as I know. See the GitHub's related page https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project

Reply:
Github suggest 3 places root, docs, .github. I prefer to have one place for all github related docs, and prefer it not to be the root as it has many unrelated files.

From your link: 
To make your code of conduct visible in the repository's root directory, type CODE_OF_CONDUCT in the file name field.
To make your code of conduct visible in the repository's docs directory, type docs/CODE_OF_CONDUCT.
To make your code of conduct visible in the repository's .github directory, type .github/CODE_OF_CONDUCT.

Reply:
@IgorGov oh yes correct, I'm sorry. I'm just too used to seeing it almost always in the root directory but OK. Moved it to `.github/` directory.